Example/Sources/AppDelegate.swift

```diff
@@ -38,10 +38,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		// For using the Order API, Virtusize.userID is required
 		Virtusize.userID = "123"
 		// By default, the Virtusize environment will be set to .global
-		Virtusize.environment = .staging
+		Virtusize.environment = .JAPAN
```

Comment:
I think we can also modify the comment above by: 
By default, the Virtusize environment will be set to **.global** → By default, the Virtusize environment will be set to **.JAPAN**

Reply:
Comment said if we are not setting any environment, By default .GLOBAL will be best set so we do not need to change comment

Reply:
OK Thanks